sdk/eventhub/event-hubs/src/eventProcessor.ts

```diff
@@ -233,7 +241,7 @@ export class EventProcessor {
     this._processorOptions = options;
     this._pumpManager = new PumpManager(this._id, this._processorOptions);
     const inactiveTimeLimitInMS = 60000; // ownership expiration time (1 mintue)
-    this._partitionLoadBalancer = new PartitionLoadBalancer(this._id, inactiveTimeLimitInMS);
+    this._partitionLoadBalancer = options.partitionLoadBalancer || new FairPartitionLoadBalancer(this._id, inactiveTimeLimitInMS);
```

Comment:
Might be worth testing with the partition manager from eventhubs-checkpointstore-blob if you have a chance.

Reply:
I'll run a test but...my change should not have changed anything there since all the events/hooks are the same.